crates/bindings-csharp/Runtime/Internal/Module.cs

```diff
@@ -411,11 +424,20 @@ public static Errno __call_view_anon__(uint id, BytesSource args, BytesSink rows
     }
 }
 
+/// <summary>
+/// Read-write database access for procedure contexts.
+/// The code generator will extend this partial class with table accessors.
+/// </summary>
+public partial class Local
+{
+    // Intentionally empty – generated code adds table handles here.
+}
+
 /// <summary>
 /// Read-only database access for view contexts.
 /// The code generator will extend this partial class to add table accessors.
 /// </summary>
-public sealed partial class LocalReadOnly
```

Comment:
Why did this change? `LocalReadOnly` is for views, which should be unrelated to this procedure PR, right?